base-template.sh

```diff
@@ -760,15 +760,17 @@ show_prompt() {
 
     # Try to read from `dev/tty` if available.
     # Our stdin is never a tty (either a pipe or /dev/null when called
-    # from git), so read from /dev/tty, our controlling terminal.
-    # However, only do this when stdout *is* a tty, otherwise it is
-    # likely we have no controlling terminal and reading from /dev/tty
-    # would fail with an error.
+    # from git), so read from /dev/tty, our controlling terminal,
+    # if it can be opened.
     printf "%s %s [%s]:" "$TEXT" "$HINT_TEXT" "$SHORT_OPTIONS"
-    # shellcheck disable=SC2229
-    read -r "$VARIABLE" </dev/tty 2>/dev/null
+    #shellcheck disable=2217
```

Comment:
```suggestion
    # shellcheck disable=SC2217
```

For consistency.

Reply:
Looks good otherwise, thanks @gabyx !
Could you please apply the suggestion, then I'll merge on the green Travis build. Thanks!

Reply:
:+1:  Done. Regex replaced all of them which where not as you wish.